### DISCUSSION TITLE
Is physical world snapshot and recovery supported? What do I need to do?

### DISCUSSION THREAD

**Discussion by PinkFatMan** [DanielChappuis/reactphysics3d#352] [Q&A]
Hi, I'm working on the reconnection mechanic for the game. I wanted to serialize the entire physical world at key points in the game and save it to a local file system. Read serialized files on equal reconnection and quickly restore the game world. May I ask what I need to pay attention to in order to realize this idea

**Comment by DanielChappuis** [comment]
Hello,

The ReactPhysics3D library does not support serializing the physics world at the moment. You will need to do it manually.

Note that because you wont be able to serialize internal state of the physics simulation, restoring a serialization of the world and continuing the simulation (using PhysicsWorld::update()) might not perfectly continuous.
